scripts/consts.ts

```diff
@@ -0,0 +1,24 @@
+import { Address } from 'viem'
```

Comment:
can you think of a better filename than "consts"?  How about "networks" or something?

Reply:
Changed to `networks.ts` and `filters.ts` in af01c0a2eb8b761c9588f99d9bc0cad8989f2178.